griptape/drivers/prompt/openai_chat_prompt_driver.py

```diff
@@ -79,7 +92,25 @@ def try_run(self, prompt_stack: PromptStack) -> TextArtifact:
         parsed_result = result.parse()
 
         if len(parsed_result.choices) == 1:
-            return TextArtifact(value=parsed_result.choices[0].message.content.strip())
+            message = parsed_result.choices[0].message
+            tool_calls: list[ChatCompletionMessageToolCall] = message.tool_calls
+
+            if tool_calls:
+                actions = [
+                    ActionsArtifact.Action(
+                        tag=tool_call.id,
+                        name=tool_call.function.name.split("-")[0],
+                        path=tool_call.function.name.split("-")[1],
```

Comment:
Why do you put these split these and put them into `name` and `path` rather than just putting `function.name` directly into `name`?